blocks/library/freeform/test/__snapshots__/index.js.snap

```diff
@@ -8,7 +8,7 @@ Array [
     style="display:none"
   />,
   <div
-    class="wp-block-freeform blocks-rich-text__tinymce"
```

Comment:
Wondering why this change, you didn't touch the `edit` function right?

Reply:
>Wondering why this change, you didn't touch the `edit` function right?

Because I added the `supports.className: false`, it's no longer adding the generated class in `BlockEdit`:

https://github.com/WordPress/gutenberg/blob/0cbe106b28270694515930303a8162c5d846bdd0/blocks/block-edit/index.js#L28-L32

...though this makes me think that this is a bit hard to follow, and that we might want to consolidate class name handling into `blocks/hooks/generated-class-name.js`, particularly since `BlockEdit` is already filterable anyways.

Reply:
Oh right I confused it with `customClassName`